features/api.feature

```diff
@@ -630,11 +630,50 @@ Feature: Browsing via API
 
   Scenario: Download items metadata and files in Zip format including non-existent items
     Given I ingest "cooee:1-001"
-    Given I make a JSON post request for the download_items page with the API token for "researcher1@intersect.org.au" with JSON params
+    When I make a JSON post request for the download_items page with the API token for "researcher1@intersect.org.au" with JSON params
       | format | items                                                                                    |
       | zip    | ["http://example.org/catalog/cooee/1-001","http://example.org/catalog/cooee/non-exists"] |
     Then I should get a 200 response code
 
+  Scenario: Download items metadata and files in Zip format with a bad regular expression
+    Given I ingest "cooee:1-001"
+    When I make a JSON post request for the download_items page with the API token for "researcher1@intersect.org.au" with JSON params
+      | format | items                                                                               | regexp |
+      | zip    | ["http://example.org/catalog/cooee/1-001","http://example.org/catalog/cooee/1-002"] | *.txt  |
+    Then I should get a 400 response code
+    And the JSON response should be:
+    """
+    {"error":"Error with regular expression: target of repeat operator is not specified: /*.txt/"}
```

Comment:
You are right - but regex and glob are used interchangeably.
I honestly think that if you demonstrate this story in it's current state to the client then it's going to come back for rework because most people think a file-selecting regex should look like *.txt rather than .*\.txt$